executor/show_stats_test.go

```diff
@@ -342,18 +342,6 @@ func TestShowColumnStatsUsage(t *testing.T) {
 	require.Equal(t, rows[1], []interface{}{"test", "t2", p0.Name.O, t1.Meta().Columns[0].Name.O, "2021-10-20 09:00:00", "<nil>"})
 }
 
-func TestShowHistogramsInFlight(t *testing.T) {
```

Comment:
Why remove this test?

Reply:
Bazel isolated the test by the folder. if too many tests are in the same folder, it will have many problems. so we need to move it into another package in a different folder.

Reply:
> many problems

Could you elaborate a bit on this statement?

Reply:
I don't know why it happens. but I try to remove it into another folder to keep the tests independent. it is successful.

Reply:
Fine. Then let's remain as is.